### ISSUE TITLE
[BUG] Plugin is not fully compatible with Angular 13

### ISSUE THREAD

**Issue by nseni** [bithost-gmbh/ngx-mat-select-search#347]
**Describe the bug**
When I build my app I get this message:
```
Processing legacy "View Engine" libraries:
- ngx-mat-select-search [es2015/esm2015] (https://github.com/bithost-gmbh/ngx-mat-select-search.git)
Encourage the library authors to publish an Ivy distribution.
```

**Additional context**
```
Angular CLI: 13.0.3
Node: 16.13.0
Package Manager: npm 8.1.0
OS: win32 x64

Angular: 13.0.2
... animations, cdk, common, compiler, compiler-cli, core, forms
... material, platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1300.3
@angular-devkit/build-angular   13.0.3
@angular-devkit/core            13.0.3
@angular-devkit/schematics      13.0.3
@angular/cli                    13.0.3
@schematics/angular             13.0.3
rxjs                            7.4.0
typescript                      4.4.4
---------------------------------------------------------
ngx-mat-select-search           3.3.3
```

**Comment by macjohnny** [comment]
Thanks for reporting, will update the package soon

**Comment by Odonno** [comment]
This library is now indeed Angular 13 compatible but unfortunately it still isn't using Ivy distribution.

**Comment by macjohnny** [comment]
fixed in version 4.0.0
